app/services/add_assigned_scope_limitation_service.rb

```diff
@@ -0,0 +1,28 @@
+class AddAssignedScopeLimitationService
+  def initialize(legal_aid_application, proceeding_type_id, scope_type)
+    @legal_aid_application = legal_aid_application
```

Comment:
this isn't being used anywhere is it? There's no attr_reader either so don't see how another outside of this would use it either.

Reply:
When you say this, do you mean the line `@legal_aid_application = legal_aid_application`? If so we use it in the next line along with proceeding_type_id to get the specific proceeding. Let me know if I have misunderstood.

Reply:
I mean, couldn't you just ditch line 3 and on line 4 do
```
  @application_proceeding_type = legal_aid_application.application_proceeding_types.find_by(proceeding_type_id: proceeding_type_id)
```
without the need to make it an instance variable?

Reply:
I misunderstood, yes good idea.